### PR TITLE
Fix HTML validation of login form

### PR DESCRIPTION
**Pull request by NiklasBr**
Fix most *info*, *warning*, and *errors* reported by: https://validator.w3.org/nu/?showsource=yes&showoutline=yes&doc=https%3A%2F%2Fdemo.pimcore.fun%2Fadmin%2Flogin

### CODE REVIEW

**Comment by robertSt7** [comment]
Hi @NiklasBr could you please fix the conflicts? Thanks

File: templates/admin/login/login.html.twig [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% extends '@PimcoreAdmin/admin/login/layout.html.twig' %}
 
 {% block content %}
-<div id="loginform">
+<div id="loginform" {% if not browserSupported %}style="display:none;"{% endif %}>
```
Comment (robertSt7) [inline templates/admin/login/login.html.twig:4]: @NiklasBr Why do you move the css to the inline style?

Comment (NiklasBr) [reply templates/admin/login/login.html.twig:4]: image

Comment (robertSt7) [reply templates/admin/login/login.html.twig:4]: @NiklasBr What is your opinion about adding this here? https://github.com/pimcore/admin-ui-classic-bundle/pull/227/files#diff-55e6dc29c563c3b12119df18461f947de2b3c73c6dae2cd25a0a2400f0e64ca6L52

Comment (NiklasBr) [reply templates/admin/login/login.html.twig:4]: I think it spreads out the logic in an unnecessary way, but that is not a strongly held opinion.

Comment (robertSt7) [reply templates/admin/login/login.html.twig:4]: We have the JS part also there, do you see any reason why this shouldn't be together?

Comment (NiklasBr) [reply templates/admin/login/login.html.twig:4]: Because this PR was meant to only fix validation errors and not become an everythingburger :)

Comment (robertSt7) [reply templates/admin/login/login.html.twig:4]: I totally agree with you, but I don't get the difference of change between moving it to the inline styling or moving it to the `below_footer` block. So you basically need to move it with both approaches, right?

Comment (blankse) [reply templates/admin/login/login.html.twig:4]: @robertSt7 `below_footer` is also inside of the body tag. You would need a new block for the head section.

Comment (robertSt7) [reply templates/admin/login/login.html.twig:4]: @blankse You are right. Thanks 
@NiklasBr Then we would consider this in a follow up PR

**Comment by robertSt7** [comment]
@NiklasBr Thanks for the PR :)

**Pull request merged**